doc/i18n.md

```diff
@@ -0,0 +1,71 @@
+
+Cockpit Internationalization
+============================
+
+All human-visible strings on Cockpit's web pages should be translatable. This
+document describes the entire internationalization process, which uses [GNU gettext](https://www.gnu.org/software/gettext/) and related tools as much as possible.
+
+Marking strings
+---------------
+Strings that are visible to humans and which should be translated must be
+marked as such on Cockpit pages, in one of two ways:
+
+ * If the string appears in a HTML source, it must be enclosed in a tag with
+   the `translatable="yes"` attribute. Example:
+
+   ```html
+   <span translatable="yes">Image</span>
```

Comment:
This should just say ```translate```.

```
<span translate>Image</span>
```

Reply:
The documentation of `cockpit.translate()` says "translatable", and the vast majority of pages (all except playground, basically)  use `translatable`.  OTOH the "official" tag [seems to be](https://www.w3.org/International/questions/qa-translate-flag) `translate`. `cockpit.translate()` actually recognizes both, but if we go with the latter, should we first mass-rename our existing pages to avoid confusion?

Reply:
We support both. We should just document that, with a note that new code should use ```translate```.

Reply:
Agreed, makes sense. I changed it to `translate` now and also updated the documentation of `cockpit.translate()`.